src/lib/models/user.ts

```diff
@@ -0,0 +1,11 @@
+import { auth } from '@clerk/nextjs/server'
+
+export type AuthUser = {
+  id: string
+}
+
+export const getCurrentAuthUser = (): AuthUser => {
```

Comment:
I create this interface so that if we ever need to modify or swap out Clerk interaction, there is just one place to do it 😄